config.yaml

```diff
@@ -1,16 +1,28 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file defines charm config options, and populates the Configure tab on Charmhub.
-# If your charm does not require configuration options, delete this file entirely.
-#
-# See https://juju.is/docs/config for guidance.
 
 options:
-  # An example config option to customise the log level of the workload
-  log-level:
-    description: |
-      Configures the log level of gunicorn. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
-    default: "info"
+  host:
+    description: The hostname or IP address of the outgoing SMTP relay
+    type: string
+  port: 
+    description: The port of the outgoing SMTP relay
+    type: int
+    default: 25
+  user:
+    description: The SMTP AUTH user to use for the outgoing SMTP relay
+    type: string
+  password:
+    description: The SMTP AUTH password to use for the outgoing SMTP relay
+    type: string
+  auth_type:
+    description: The type used to authenticate with the SMTP relay. It has to be one of "none", "not_provided" or "plain"
+    type: string
+    default: "none"
```

Comment:
Out of curiosity, I'm wondering the reason for including "none" as one of the options rather than leaving the field empty?